pkg/cfg/error.go

```diff
@@ -1,37 +1,37 @@
 package cfg
 
 import (
+	"fmt"
 	"github.com/getsentry/sentry-go"
-	"github.com/pkg/errors"
 )
 
 //go:generate mockery -name Sentry
 type Sentry interface {
 	CaptureException(exception error, hint *sentry.EventHint, scope sentry.EventModifier) *sentry.EventID
 }
 
-type ErrorHandler func(err error, msg string, args ...interface{})
+type ErrorHandler func(msg string, args ...interface{})
 
 var defaultErrorHandler = PanicErrorHandler
 
 func WithDefaultErrorHandler(handler ErrorHandler) {
 	defaultErrorHandler = handler
 }
 
-func PanicErrorHandler(err error, msg string, args ...interface{}) {
-	err = errors.Wrapf(err, msg, args...)
+func PanicErrorHandler(msg string, args ...interface{}) {
+	err := fmt.Errorf(msg, args...)
 	panic(err)
 }
 
 func LoggerErrorHandler(logger Logger) ErrorHandler {
-	return func(err error, msg string, args ...interface{}) {
-		logger.Error(err, msg, args...)
+	return func(msg string, args ...interface{}) {
+		logger.Error(msg, args...)
 	}
 }
 
 func SentryErrorHandler(sentry Sentry) ErrorHandler {
-	return func(err error, msg string, args ...interface{}) {
-		err = errors.Wrapf(err, msg, args...)
+	return func(msg string, args ...interface{}) {
+		err := fmt.Errorf(msg, args...)
```

Comment:
I wonder... `errors.Wrapf` adds a stack trace somehow... do we need this? Does sentry add its own stack trace?

Reply:
Luckily the `SentryErrorHandler` is not used anywhere, so we don't have to care about this here..  
But in general the sentry stuff needs some overhaul, as it uses quite some old error packages.  
This is planned for the incoming new sentry handler for the incoming new logger package..